ch_backup/logic/lock_manager.py

```diff
@@ -86,6 +98,9 @@ def __exit__(
             self._zk_lock.release()
             self._zk_client.__exit__(None, None, None)
 
+        logging.debug(self._log_msg_with_lock_id("Exiting lock."))
+        self._lock_id = None
```

Comment:
It looks suspicious. Where does this help, given that we always set `_lock_id` in the `__call__`. And now we can't do that:
```
locker = locker(lock_id="id")
with locker:
 # do smth
...
with locker:
 # do smth
```

Reply:
yes, you are right. It could be redundant. I just wanted to be sure that an lock id relative to an old operation was not left as _lock_id value. If we can consider this never happens, it makes thinks simpler.